src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -203,31 +219,43 @@ public AudioData generateAudioData(AudioData audioData, String toSpeak) throws I
     return audioData;
   }
 
+  public boolean setSsml(boolean ssml) {
+    PollyConfig c = (PollyConfig) config;

```

Comment:
this shouldn't reference the config here...  please remove the config as a member for these classes.